packages/shared/useArrayUnique/index.md

```diff
@@ -33,6 +33,24 @@ const list = reactive([1, 2, 2, 3])
 const result = useArrayUnique(list)
 // result.value: [1, 2, 3]
 
-result.value.push(1)
+result.push(1)
```

Comment:
What's the reason of this change?

Reply:
Sorry here should be list.push I have modified it in the latest commit